dali/base/dacsds.ipp

```diff
@@ -333,8 +335,8 @@ public:
     virtual void appendLocal(size32_t size, const void *data, bool binary);
     virtual void addingNewElement(IPropertyTree &child, int pos);
     virtual void removingElement(IPropertyTree *tree, unsigned pos);
-    virtual void setAttr(const char *attr, const char *val);
-    virtual bool removeAttr(const char *attr);
+    virtual void setAttribute(const char *attr, const char *val);
```

Comment:
yep, I like override, it's saved me in the past. TBH I'd forgotten to use it recently.